### DISCUSSION TITLE
zing GC_active confuse

### DISCUSSION THREAD

**Discussion by liujingjun001** [async-profiler/async-profiler#968] [Q&A] [answered]
the question is:
1. what is GC_active? is async-profiler's bug? 
2. is GC_active has any value for performance optimization?

**Comment by apangin** [comment, marked as answer]
The question is a duplicate of https://github.com/async-profiler/async-profiler/issues/969

`[GC_active]` means the JVM could not get a stack trace because a concurrent GC was running. This is not an async-profiler bug.
High `[GC_active]` percentage likely means that the heap size is too small and/or application allocation rate is too high.
